src/Models/Stat.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Spatie\Stats\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Stat extends Model
+{
+
+    public function events()
+    {
+        return $this->hasMany(StatsEvent::class);
+    }
+}
```

Comment:
Is this model only used for tests?

Reply:
Yes this is used only for tests. I didn't had an idea how to create a model (+ migration) only for tests.

Reply:
@AlexVanderbist should I do this differently? Are you aiming for a different approach here?

Reply:
That worked out like a breeze and I learned something new. Thank you! 🙏  I guess we're good to go now!